draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1001,10 +1005,10 @@ cookies" (either a registrable domain, or the empty string):
     1.  Let `origin` be the origin of `item`'s URI if `item`'s sandboxed origin
         browsing context flag is set, and `item`'s origin otherwise.
 
-    2.  If `origin`'s host's registrable domain is not an exact match for
-        `top-origin`'s host's registrable domain, return the empty string.
+    2.  If `origin` is not same-site to `top-origin`, return an origin set to a 
```

Comment:
I dunno, is it "same-site to" or "same-site with"?

Reply:
Let's go with "with", since that's used elsewhere in the document.